src/test/fixtures/composes.tsx

```diff
@@ -232,124 +246,298 @@ export const mockComposes = {
   ],
 };
 
-export const mockStatus = {
-  '1579d95b-8f1d-4982-8c53-8c2afa4ab04c': {
-    image_status: {
-      status: 'success',
-      upload_status: {
-        options: {
-          ami: 'ami-0217b81d9be50e44b',
-          region: 'us-east-1',
-        },
+export const mockStatus = (composeId: string): ComposeStatus => {
+  const mockComposes = {
```

Comment:
We can actually type this object when it is declared, rather than typing the individual keys:
```suggestion
  const mockComposes: { [key: string]: ComposeStatus } = {
```

Reply:
Ah, got it. Looks much cleaner. Updated